README.md

```diff
@@ -12,6 +12,30 @@ All features are contributed by you, our amazing .NET community, and maintained
 
 And – the best part – the features you add to the .NET MAUI Toolkit may one day be included into the official .NET MAUI library! We leverage the Community Toolkits to debut new features and work closely with the .NET MAUI engineering team to nominate features for promotion.
 
+## Getting Started
+
+In order to use the .NET MAUI Community Toolkit you need to call the extension method in your `MauiProgram.cs` file as follows:
+
+```csharp
+public static MauiApp CreateMauiApp()
+{
+    var builder = MauiApp.CreateBuilder();
+
+    // Initialise the toolkit
+	builder.UseMauiApp<App>().UseMauiCommunityToolkit();
+
+    // the rest of your logic...
+}
+```
+
+### XAML usage
+
+In order to make use of the toolkit within XAML you can use this namespace:
+
+```xml
+xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
```

Comment:
Let's recommend using `toolkit` instead of `mct`.

Devs can use whichever keyword they prefer, but, long story short, we'd get in trouble with marketing/legal for recommending `mct`.

```suggestion
xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
```